example/winrt_picker.dart

```diff
@@ -2,74 +2,38 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// File Open Picker from Dart
+// Demonstrates calling WinRT FileOpenPicker to pick a file from console window.
 
-import 'dart:ffi';
-
-import 'package:ffi/ffi.dart';
 import 'package:win32/winrt.dart';
 
 void main() async {
   winrtInitialize();
-  final pIndex = calloc<Uint32>();
 
   final picker = FileOpenPicker()
     ..suggestedStartLocation = PickerLocationId.desktop
```

Comment:
I'm not sure why, but neither this nor the `viewMode` property appear to be working on my machine. I've tried changing them to various other values, but they don't seem to affect the presented dialog box. Can you replicate?

Reply:
[FileOpenPicker.SuggestedStartLocation Property](https://learn.microsoft.com/en-us/uwp/api/windows.storage.pickers.fileopenpicker.suggestedstartlocation?view=winrt-22621#remarks) states:
> The SuggestedStartLocation is not always used as the start location for the file picker. To give the user a sense of consistency, the file picker remembers the last location that the user navigated to and will generally start at that location.

As for the `viewMode` property, I can't get it to work either. I'm installing Visual Studio right now. I'll try this example in a UWP app to see if this works properly there.

Reply:
Interesting, it doesn't work in a UWP app either.

Reply:
Very strange. Investigating here too...